external/miniz/Kconfig

```diff
@@ -3,7 +3,7 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-config COMPRESSION_MINIZ
+config LIBMINIZ
 	bool "Miniz Compression"
```

Comment:
How about "Miniz library" for the same description with LZMA?

Reply:
yes, will add in new PR with miniz cleanup